hack/images/titus-sshd/run-titus-sshd

```diff
@@ -0,0 +1,41 @@
+#!/titus/sshd/bin/busybox sh
+
+write_etc_profile() {
+cat > /etc/profile <<'EOF'
```

Comment:
Dumb question, but these are all binaries sourced from the prefix, and not from the user container, right?

Reply:
Great question. Let me test how the path is constructed here. 
Busybox will put its binaries in the end of the path, so I think yea, I should use full explicit paths so we never accidentally run user code here.

Reply:
`/etc` is getting created by the executor, so we don't need to `mkdir -p` it, correct?

Reply:
correct